content/requests/sign_request_create_request.yml

```diff
@@ -0,0 +1,158 @@
+---
+title: Create a sign request
+
+type: object
+
+description: |-
+  A request to create a sign request object
+
+required:
+  - signers
+  - source_files
+
+properties:
+  is_document_preparation_needed:
+    type: boolean
+    description: >-
+      Indicates if the sender should be taken into the builder flow to
+      prepare the document.
+    example: true
+
+  redirect_url:
+    type: string
+    description: The uri that a signer will be redirect to after signing a
+      document. If no declined redirect url is specified, this will be
+      used for decline actions as well. 
+    example: https://redirect.com
+
+  declined_redirect_url:
+    type: string
+    description: >-
+      The uri that a signer will be redirect to after declining to sign a
+      document
+    example: https://declined-redirect.com
+
+  required_attachments:
+    type: array
+    items:
+      type: object
+      properties:
+        name:
+          type: string
+          example: passport
+          description: >-
+            name of attachment to be uploaded by signer
+    description: Attachments that signers are required to upload
+
+  are_attachments_enabled:
+    type: boolean
+    description: Disable uploading/adding attachments for signers
+    example: true
+    default: true
+
+  are_text_signatures_enabled:
+    type: boolean
+    description: Disable usage of signatures generated by typing (text)
+    example: true
+    default: true
+
+  is_text_enabled:
+    type: boolean
+    description: Disable ability for signer to add text
+    example: true
+    defaut: true
+
+  are_dates_enabed:
+    type: boolean
+    description: Disable ability for signer to add dates
+    example: true
+    default: true
+
+  are_emails_disabled:
+    type: boolean
+    description: >-
+      Disable all status emails, as well as the original email that
+      contains the sign request
+    example: false
+    defaut: false
+
+  signature_color:
+    type: string
+    enum: ['blue', 'black', 'red']
+    example: blue
+    description: >-
+      Force a specific color for the signature
+
+  is_phone_verification_required_to_view:
+    type: boolean
+    description: >-
+      Forces signers to verify a text message prior to viewing the
+      document. You must specify the phone number of signers to have
+      this setting apply to them.
+    example: true
+
+  email_subject:
+    type: string
+    example: Sign Request from Acme
+    description: Subject of sign request email. This is cleaned by sign
+      request.
+
+  email_message:
+    type: string
+    example: Hello! Please sign the document below
+    description: Message to include in sign request email. This is cleaned,but
```

Comment:
Can they change background, embed image link to company logo, have borders, etc? Seems like we need more detail than "some html tags are allowed"

Reply:
The message may contain the following html tags: a, abbr, acronym, b, blockquote, code, em, i, ul, li, ol, and strong. Be aware that when the text to html ratio is too high, the email may end up in spam filters. Custom styles on these tags are not allowed.

Reply:
^ will add the above details to the description